dags/main_summary.py

```diff
@@ -81,6 +81,44 @@
             "--partition-by=sample_id",
             "--replace='{{ds_nodash}}' AS submission_date",
             "--maps-from-entries",
+            "--bigint-columns",
+            "search_counts.count",
```

Comment:
Perhaps add a comment here about how we arrived at this list and under what circumstances we might need to update it.

Reply:
fixed